pvt/helpers/src/models/pools/weighted/types.ts

```diff
@@ -11,12 +11,69 @@ import Vault from '../../vault/Vault';
 export enum WeightedPoolType {
   WEIGHTED_POOL = 0,
   LIQUIDITY_BOOTSTRAPPING_POOL,
-  MANAGED_POOL,
-  MOCK_MANAGED_POOL,
-  MOCK_MANAGED_POOL_SETTINGS,
+}
+
+export enum ManagedPoolType {
+  MANAGED_POOL = 'ManagedPool',
+  MOCK_MANAGED_POOL = 'MockManagedPool',
+  MOCK_MANAGED_POOL_SETTINGS = 'MockManagedPoolSettings',
 }
```

Comment:
Ah, nice move here using the contract name as the enum value :+1: 

Maybe I'd add a comment that this is on purpose to make deployments easier in the helpers, since these names are used directly to fetch the artifacts.